sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.institutions.controller.exportReport.e2e-spec.ts

```diff
@@ -255,4 +260,209 @@ describe("ReportInstitutionsController(e2e)-exportReport", () => {
+            finalFederalAwardNetCSGFAmount: 18,
+            finalProvincialAwardNetBGPDAmount: 19,
+            totalAssessedCost: 50,
+          } as Assessment,
+        },
+        applicationData: {
+          indigenousStatus: "no",
+          citizenship: "canadianCitizen",
+          youthInCare: "no",
+          dependantstatus: "independant",
+        } as ApplicationData,
+      },
+    );
+    const payload = {
+      reportName: "Student_Unmet_Need_Report",
+      params: {
+        offeringIntensity: {
+          "Full Time": true,
+          "Part Time": true,
+        },
+        programYear: programYear.id,
+      },
+    };
+    const dryRunSubmissionMock = jest.fn().mockResolvedValue({
+      valid: true,
+      formName: FormNames.ExportFinancialReports,
+      data: { data: payload },
+    });
+    formService.dryRunSubmission = dryRunSubmissionMock;
+    const endpoint = "/institutions/report";
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .then((response) => {
+        const fileContent = response.request.res["text"];
+        const parsedResult = parse(fileContent, {
+          header: true,
+        });
+        expect(parsedResult.data).toEqual(
+          expect.arrayContaining([
+            {
+              "Application Disability Status": "no",
+              "Application Number": savedApplication.applicationNumber,
+              "Assessment Date": getISODateOnlyString(
+                savedApplication.currentAssessment.assessmentDate,
+              ),
+              "CIP Code":
+                savedApplication.currentAssessment.offering.educationProgram
+                  .cipCode,
+              "Citizenship Status":
+                savedApplication.currentAssessment.application.data.citizenship,
+              "Estimated BCAG": String(
+                savedApplication.currentAssessment.assessmentData
+                  .finalProvincialAwardNetBCAGAmount,
+              ),
+              "Estimated BCSL": String(
+                savedApplication.currentAssessment.assessmentData[
+                  "finalProvincialAwardNetBCSLAmount"
+                ],
+              ),
+              "Estimated BGPD": String(
+                savedApplication.currentAssessment.assessmentData[
+                  "finalProvincialAwardNetBGPDAmount"
+                ],
+              ),
+              "Estimated CSGD": String(
+                savedApplication.currentAssessment.assessmentData
+                  .finalFederalAwardNetCSGDAmount,
+              ),
+              "Estimated CSGF": String(
+                savedApplication.currentAssessment.assessmentData[
+                  "finalFederalAwardNetCSGFAmount"
+                ],
+              ),
```

Comment:
The `assessment` is defined as 
```ts
export type Assessment = FullTimeAssessment | PartTimeAssessment;
```
and the indentation is to have the cast executed when needed.
So, if we know those values are part of a full-time assessment, casting to `FullTimeAssessment` will allow typed access to the properties.